python/snewpy/scripts/Convert_to_ROOT.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+""" This module allows to gather the SNOwGLoBES outputs into a ROOT files with 2D histograms.
+    These 2D histograms show the rate as a function of time and energy for any given interaction channel.
+    There is one ROOT file per detector, per smearing option, and per weighting option.
+
+    To run the code, use:
```

Comment:
This script currently only works for tarballs generated by `generate_time_series()`, not `generate_fluence()` where the filenames are a bit different. For a sample script, I think that’s okay; but we should add a brief explanation at the start to avoid confusion.
```suggestion
    Generate a tarball with the input files by running
      snowglobes.generate_time_series(...)
      snowglobes.simulate(...)
      snowglobes.collate(...)

    Then, to run the code, use:
```

Reply:
I implemented the changes